ResearchKit/Common/ORKResult.h

```diff
@@ -1135,4 +1135,26 @@ ORK_CLASS_AVAILABLE
 
 @end
 
+/**
+ The `ORKLocationQuestionResult` class represents the result of a question or form item that asks for a location (`ORKLocationAnswerFormat`).
+ 
+ A Location question result is produced by the task view controller when it presents a question or form
+ item with a Location answer format (that is, `ORKLocationAnswerFormat`).
+ 
+ A Location question result is typically generated by the framework as the task proceeds. When the task
+ completes, it may be appropriate to serialize it for transmission to a server,
+ or to immediately perform analysis on it.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKLocationQuestionResult : ORKQuestionResult
+
+/**
+ The NSValue` object wrapped in `MKCoordinate` which repesents the latitude and longitude of the location the user entered,
+ or `nil` if the user skipped the question.
+ */
+@property (nonatomic, copy, nullable) NSValue *locationAnswer;
+
```

Comment:
Beside the `Location Coordinates`, would it be nice to include a text based address, which is human readable.